lib/cli.nix

```diff
@@ -90,7 +90,16 @@ rec {
     mkOption ?
       k: v: if v == null
             then []
-            else [ (mkOptionName k) (lib.generators.mkValueStringDefault {} v) ]
+            else if optionValueSeparator == null then
+              [ (mkOptionName k) (lib.generators.mkValueStringDefault {} v) ]
+            else
+              [ "${mkOptionName k}${optionValueSeparator}${lib.generators.mkValueStringDefault {} v}" ],
+
+    # how to separate an option from its flag;
+    # by default, there is no separator, so option `-c` and value `5`
+    # would become ["-c" "5"].
+    # This is useful if the command requires equals, for example, `-c=5`.
```

Comment:
Unfortunately this is currently not rendered in the reference manual. If you're interested in improving that, feel free to make a PR to move these into the doc comment above the function! Check out the part on 'function arguments' [here](https://github.com/NixOS/nixpkgs/tree/master/doc#documentation-conventions) to see the convention for that :)

Reply:
Thanks for the tip - would you mind taking a look at #315820? Also, do you have a link to the reference doc? I can't find where `cli.nix` appears when I've been searching